extensions/amp-analytics/0.1/vendors.js

```diff
@@ -507,6 +507,40 @@ export const ANALYTICS_CONFIG = {
       },
     },
   },
+  'oewadirect': {
```

Comment:
I know that other vendors are not sorted, but can you move these so that these are in the right place (alphabetical sorting?) Same for the second config.